### ISSUE TITLE
Manifests: highlight if short_name exceeds expected character count

### ISSUE THREAD

**Issue by addyosmani** [GoogleChrome/lighthouse#69]
Afaik on the Android homescreen, if a `short_name` contains too many characters it will be trimmed to something like `My Totally Awesome App -> My Totally Awe...`. It may be useful to check the spec or with Mounir regarding expected lengths for these app names and whether flagging this via Lighthouse makes sense.

I've personally run into issues with needing to manually test this on-device (or rethink the names used) to avoid truncating, so throwing the idea out there.

**Comment by paulirish** [comment]
Can you find out what character length range we should suggest?

**Comment by addyosmani** [comment]
Yep!

**Comment by paullewis** [comment]
@mounirlamouri might know!

**Comment by mounirlamouri** [comment]
I do not think we do any kind of significant trimming in Chrome (there is a 4Ki limit but not what you are asking I think :)).

@addyosmani where do you see the trim? Is it in the home screen? or the Chrome UI?

**Comment by paullewis** [comment]
I think it's more the visual wrangling you do to make sure it's not truncated.

**Comment by addyosmani** [comment]
@mounirlamouri I see it in the homescreen. I'll see if I can put together a repro.

**Comment by paulirish** [comment]
This is a smart check to have, yeah.

I am pretty sure i've seen quite different results on a Nexus 6 vs a smaller phone for this number.  

@addyosmani have you gotten a sense of what string lengths work?

**Comment by paulirish** [comment]
From my homescreen.. here's what is and isn't truncated, along with string lengths:

```sh
HackerWeb # 9
Play Store  # 9
Pocket Cas… # 10
Chrome Ca… # 9
Popcorn Ti… # 10
What’s my…  # 9
Flipkart Lite   # 13
Guitar Tuner # 12
Trained To… # 10
Inbox Attack # 12
Get Kana! # 9
BusRouter… #9
```

By now, you can tell that Roboto doesn't use the same width for all letters. :)
An app called `iiiiiiiiiiiiiiiii` (with 17 i's) will fit, but `wwwwwwww` (with 8 w's) won't.

**Comment by paullewis** [comment]
If we know widths and fonts we can guess pretty well with a `measureText`. We could ask the page to do it in JS as a gather once we have a manifest. It's messy.

**Comment by samccone** [comment]
Let me follow up with the android UI team... and see if I can get a real answer on this.

**Comment by paulirish** [comment]
fixed by #145
